templates/userdata.tmpl

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+/usr/local/bin/aws configure set profile.default.region eu-west-1
+/usr/local/bin/aws configure set profile.default.output json
+/usr/local/bin/aws configure set profile.ebs.role_arn $(wget -q -O - 169.254.169.254/latest/meta-data/iam/info | grep "InstanceProfileArn" | awk -F\"  '$2 == "InstanceProfileArn" {print $4}')
+/usr/local/bin/aws configure set profile.ebs.source_profile default
+instance_id=$(wget -q -O - 169.254.169.254/latest/meta-data/instance-id)
+instance_role=$(/usr/local/bin/aws ec2 describe-instances --instance-ids ${instance_id} --query 'Reservations[*].Instances[].Tags[?Key==`Role`].Value[]' --output text)
+volumes_ids=$(/usr/local/bin/aws ec2 describe-volumes --filter Name=tag:Name,Values="${instance_role}" --query 'Volumes[].[VolumeId]' --output text)
```

Comment:
this might be safer in a bash array rather than string, just in case since we are depending on IFS for the for-loop iteration